### PR TITLE
Opentelemetry metrics

### PR DESCRIPTION
**Pull request by hananiel**

### CODE REVIEW

File: src/Management/Management.sln [hunk 1]
```diff
@@ -18,11 +21,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "config", "config", "{976463
 		..\..\versions.props = ..\..\versions.props
 	EndProjectSection
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{2D0A3F62-46FA-4317-BDEB-BE9433B9EBBE}"
```
Comment (TimHess) [inline src/Management/Management.sln:21]: Was this removal intentional?

File: src/Management/Management.sln [hunk 2]
```diff
@@ -6,6 +6,9 @@ MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{B86A4082-6F42-4BFC-A1F2-83CC422AD493}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{8B4CE682-9C0B-4849-813B-E43BE098FCE2}"
+	ProjectSection(SolutionItems) = preProject
+		src\TracingCore\TracingServiceCollectionExtensions.cs = src\TracingCore\TracingServiceCollectionExtensions.cs
+	EndProjectSection
```
Comment (TimHess) [inline src/Management/Management.sln:11]:
```suggestion
```

File: src/Management/src/EndpointBase/Steeltoe.Management.EndpointBase.csproj [hunk 3]
```diff
@@ -40,6 +40,9 @@
     <PackageReference Include="Steeltoe.Extensions.Logging.DynamicLogger" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
     <PackageReference Include="Steeltoe.Management.Abstractions" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
     <PackageReference Include="Steeltoe.Management.Diagnostics" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
-    <PackageReference Include="Steeltoe.Management.OpenCensusBase" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
+    <PackageReference Include="Steeltoe.Management.OpenTelemetryBase" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
+  </ItemGroup>
+  
+  <ItemGroup>
   </ItemGroup>
```
Comment (TimHess) [inline src/Management/src/EndpointBase/Steeltoe.Management.EndpointBase.csproj:47]:
```suggestion
```

File: src/Management/src/OpenTelemetryBase/Steeltoe.Management.OpenTelemetryBase.csproj [hunk 4]
```diff
@@ -17,10 +17,16 @@
...
   </ItemGroup>
 
   <ItemGroup Condition="'$(CI_BUILD)' == 'True'">
     <ProjectReference Include="Steeltoe.Common.Abstractions" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
+    <ProjectReference Include="Steeltoe.Management.Abstractions" Version="$(SteeltoeVersion)$(SteeltoeVersionSuffix)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Metrics\Factory\" />
   </ItemGroup>
```
Comment (TimHess) [inline src/Management/src/OpenTelemetryBase/Steeltoe.Management.OpenTelemetryBase.csproj:30]:
```suggestion
```

File: src/Management/Management.sln (hunk 1 above)
Comment (hananiel) [reply src/Management/Management.sln:21]: no adding back

File: src/Management/Management.sln [hunk 5]
```diff
@@ -19,9 +19,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "config", "config", "{976463
 	EndProjectSection
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{2D0A3F62-46FA-4317-BDEB-BE9433B9EBBE}"
-	ProjectSection(SolutionItems) = preProject
-		README.md = README.md
-	EndProjectSection
 EndProject
```
Comment (TimHess) [inline src/Management/Management.sln:22]:
```suggestion
	ProjectSection(SolutionItems) = preProject	
		README.md = README.md	
	EndProjectSection
EndProject
```
